de-lib/src/main/java/org/iplantc/de/client/models/apps/integration/AppTemplateAutoBeanFactory.java

```diff
@@ -43,4 +46,22 @@
     AutoBean<SimpleServiceError> simpleServiceError();
 
     AutoBean<AnalysisSubmissionResponse> analysisSubmissionResponse();
+
+    /**
+     * A method that builds an AppTemplate with default values.
+     * @return  AutoBean<AppTemplate>
+     */
+    default AutoBean<AppTemplate> defaultAppTemplate() {
+        //still cannot inject here
+        AppTemplateAutoBeanFactory factory = GWT.create(AppTemplateAutoBeanFactory.class);
+        AppTemplate newAppTemplate = factory.appTemplate().as();
```

Comment:
Instead of creating a new factory here with `GWT.create`, could we just use `AppTemplate newAppTemplate =  appTemplate().as()`? Since this a default method on this `AppTemplateAutoBeanFactory`?